lib/sprangular.rb

```diff
@@ -6,21 +6,21 @@
 require 'slim'
 require 'rabl'
 require 'jquery-rails'
+require 'rails-assets-underscore'
+require 'rails-assets-underscore.string'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.